### PR TITLE
Updated Texts

### PR DESCRIPTION
**Pull request by tichys**
changed lentext to length

added water that is sluggish to move through

### CODE REVIEW

File: code/datums/components/storage/concrete/pockets.dm [hunk 1]
```diff
@@ -56,7 +56,9 @@
 		/obj/item/scalpel, /obj/item/reagent_containers/syringe, /obj/item/dnainjector,
 		/obj/item/reagent_containers/hypospray/medipen, /obj/item/reagent_containers/dropper,
 		/obj/item/implanter, /obj/item/screwdriver, /obj/item/weldingtool/mini,
-		/obj/item/firing_pin, /obj/item/throwing_star/throwingknife
+		/obj/item/firing_pin, /obj/item/throwing_star/throwingknife, /obj/item/gun/ballistic/revolver/police, /obj/item/gun/ballistic/revolver/sequoia,
```
Comment (Celyne-git) [inline code/datums/components/storage/concrete/pockets.dm]: >sequoia

also this is gonna conflict with minortweaks

Comment (tichys) [reply code/datums/components/storage/concrete/pockets.dm]: editted to get in compliance with your tweaks

File: code/modules/clothing/suits/armor.dm [hunk 2]
```diff
@@ -417,6 +417,79 @@
...
+	name = "scout armor lt"
+	desc = "An old military grade pre war combat armor for the legatus."
+	icon_state = "scout_armor_lt"
+	item_state = "scout_armor_lt"
+	body_parts_covered = CHEST|GROIN|LEGS|ARMS
+	armor = list("melee" = 45, "bullet" = 55, "laser" = 45, "energy" = 45, "bomb" = 55, "bio" = 65, "rad" = 10, "fire" = 60, "acid" = 20)
+	strip_delay = 60
+
+/obj/item/clothing/suit/armor/f13/caeserpelt
+	name = "Caesers' Pelt"
```
Comment (Celyne-git) [inline code/modules/clothing/suits/armor.dm]: "caeser"

**Comment by Celyne-git** [comment]
i dont know what the fuck is going on with this PR but i love it.

**Pull request merged**